README.md

```diff
@@ -117,9 +117,9 @@ is used in the [Community managed DockerHub image](https://hub.docker.com/p/apac
 ## Getting started
 
 Visit the official Airflow website documentation (latest **stable** release) for help with
-[installing Airflow](https://airflow.apache.org/docs/apache-airflow/stable/installation.html),
+[installing Airflow](https://airflow.apache.org/docs/apache-airflow/stable/installation/),
 [getting started](https://airflow.apache.org/docs/apache-airflow/stable/start.html), or walking
-through a more complete [tutorial](https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html).
+through a more complete [tutorial](https://airflow.apache.org/docs/apache-airflow/stable/tutorial/).
```

Comment:
At one point in time (2.3 and earlier), this pointed at "the" tutorial. But it now points at the list of tutorials, which is better anyways!